tdrs-backend/tdpservice/users/serializers.py

```diff
@@ -54,7 +54,15 @@ class Meta:
             "first_name",
             "last_name",
             "access_request",
-            "account_approval_status"
+            "account_approval_status",
+            # Added fields
+            'groups',
+            'is_superuser',
+            'is_staff',
+            'login_gov_uuid',
+            'hhs_id',
```

Comment:
probably goes into a deeper api permissions question but i wonder if it'd be inadvisable to return some of this info via the api, since it would technically be exposed in the frontend. more of a discussion point than a callout

Reply:
> probably goes into a deeper api permissions question but i wonder if it'd be inadvisable to return some of this info via the api, since it would technically be exposed in the frontend. more of a discussion point than a callout

good discussion q @jtimpe im curious about this.  i think ideally only the sys admin should be able to make these queries. and only the sys admin has access to the api (i think? we should verify). if we need to limit what is returned, group permissions and is_staff would be most helpful in terms of permissions.

Reply:
I can hit the endpoints when logged in as a non admin user. So we have three options: scope creep this ticket and add the custom permissions, remove the sensitive fields until permissions are added, merge for the time being and push a new permissions ticket onto a high point in the priority queue. What are your thoughts: @ADPennington, @andrew-jameson, @jtimpe, and @raftmsohani?

Reply:
> I can hit the endpoints when logged in as a non admin user. So we have three options: scope creep this ticket and add the custom permissions, remove the sensitive fields until permissions are added, merge for the time being and push a new permissions ticket onto a high point in the priority queue. What are your thoughts: @ADPennington, @andrew-jameson, @jtimpe, and @raftmsohani?

couple of questions team @elipe17 @jtimpe @raftmsohani @andrew-jameson :
- does hitting the endpoint here result in a 200? 
- what would the lift be to add the custom permission?